doc/plugins/notebook.rst

```diff
@@ -0,0 +1,89 @@
+###############
+Spyder Notebook
+###############
+
+**Spyder-notebook** is a plugin that allows you to open, edit and interact with Jupyter Notebooks right inside Spyder.
+
+.. image:: /images/notebook/notebook-standard.png
+   :alt: Spyder Notebook in Spyder
+
+Using notebooks inside Spyder allows you to take advantage of their web interface alongside Spyder’s powerful features such as the Variable explorer, console and debugger.
+
+=======================
+Installing the Notebook
+=======================
+
+If you installed Spyder using conda, the best way to install Spyder-notebook is to run the following command in your terminal or Anaconda prompt on Windows:
+
+.. code-block:: bash
+
+   conda install spyder-notebook -c spyder-ide
+
+.. important::
+
+   At the moment it is not possible to use this plugin with the Spyder :ref:`standalone_installers_ref` for Windows and macOS. We're working to make that possible in the future.
+
+Restart Spyder in order to be able to use the plugin.
+
+==================
+Using the Notebook
+==================
+
+When the Notebook is installed, it will be available under the menu item :menuselection:`View --> Panes --> Notebook`.
+
+.. image:: /images/notebook/notebook-view-panes.png
+   :alt: Spyder showing view panes Notebook
+
+You will see it then as a tab in the bottom of the editor area. When switching to it, a welcome screen will be displayed, from where you can create a new notebook by right-clicking it and selecting :guilabel:`New notebook`.
+
+.. image:: /images/notebook/notebook-new-notebook-option.png
```

Comment:
I think it'd be nice to highlight with a red square or circle the notebook tab in this screeshot because lot of people miss it, like this

![imagen](https://user-images.githubusercontent.com/365293/125839301-50c60914-d9d4-45d6-80bc-07fcb8958ad4.png)